README.md

```diff
@@ -324,26 +324,10 @@ handling your report.
 
 ## Current Project Team Members
 
-The Node.js project team comprises a group of core collaborators and a sub-group
-that forms the _Technical Steering Committee_ (TSC) which governs the project. For more
-information about the governance of the Node.js project, see
-[GOVERNANCE.md](./GOVERNANCE.md).
-
-### TSC (Technical Steering Committee)
```

Comment:
You're right. Either this should be left in, or the TSC members should be included in the list of collaborators.